om-2.0.rdf

```diff
@@ -29002,6 +29026,16 @@
   <!-- Micromole Per Second Gram Ontology -->
   <!-- Nanomole Per Hour Millilitre Ontology -->
 
+  <!-- Particles Ontology -->
+
+  <om:Unit rdf:about="&om;particles">

```

Comment:
Here, too, we need clarification that this is referring to the _number_ of particles.